pycarol/apps.py

```diff
@@ -244,3 +263,64 @@ def edit_manifest(self, app_name, manifest):
 
         r = self.carol.call_api(url, method='PUT', data=manifest)
         return r
+
+
+    def get_git_process(self, app_name=None):
+        """
+        Get Git processes definid in the manifest file.
+
+        Args:
+
+            app_name: `str`
+                App name.
+
+        Returns: List of Dicts
+
+        """
+
+        if app_name is None:
+            app_name = self.carol.app_name
+
+        response = self.carol.call_api(path=f'v1/compute/{app_name}/getProcessesByGitRepo',
+                                       method='POST')
+
+        return response
+
+    def build_docker_git(self, git_token, app_name=None, ):
+        """
+        Build App image using manifest definition.
+
+        Args:
+            git_token: `str`
+                Git token to be used to pull the files.
+
+            app_name: `str`
+                App name.
+
+        Returns:
+
+        """
+
+        if app_name is None:
+            app_name = self.carol.app_name
+
+        manifest = self.get_git_process(app_name)
+
+        tasks = []
+        for build in manifest:
+            docker_name = build['dockerName']
+            docker_tag = build['dockerTag']
+            instance_type = build['instanceType']
+            git_token = git_token
+            params = {
+                'dockerName': docker_name,
+                'tagName': docker_tag,
+                'gitToken': git_token,
+                'instanceType': instance_type,
+            }
+            response = self.carol.call_api(path=f'v1/compute/{app_name}/buildGitDocker',
+                                           method='POST', params=params)
+
+            tasks.append(response)
+            
+        return response
```

Comment:
that's ok return Node here? what if None on manifest variable?

Reply:
in theory, this should not happen.. it should break in carol side.. the problem is that I was doing some tests and Carol is not validation the manifest sent by the user. Already opened a card for BE.

 I don't think this control should be on our side, what do you think @gzamboni ?

Reply:
well... If you expect a specific type from an external API, i think u should raise a exception about it, not let another expression deal with it. 
This cloud help the developer to debug in case of the external API returns a not expected response.

Reply:
I got it.. will do that..